src/windows/wslasdk/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(SOURCES wsla.cpp)
+set(HEADERS wsla.h)
+
+add_library(wslasdk SHARED ${SOURCES} ${HEADERS} wslasdk.def)
```

Comment:
I was originally of having the methods inside the `wslaclient` target, since it's essentially going to be the same thing conceptually. 

I don't mind renaming it though, but if we do that let's delete the old target